packages/contextual-menu-react/src/ContextualMenu.tsx

```diff
@@ -263,6 +263,11 @@ const ContextualMenuComponent = forwardRef<HTMLButtonElement, ContextualMenuProp
 });
 ContextualMenuComponent.displayName = "ContextualMenuComponent";
 
+/**
+ * @deprecated Denne komponenten bør ikke brukes lenger, og vil ikke bli oppdatert.
+ * Bruk heller komponenten `Menu` som er erstatning for ContextualMenu
+ */
+
```

Comment:
Denne tomme linja må fjernes for at deprecation-meldingen skal bli med i de bygde filene

```suggestion
```